src/solver/conv_winoRxS.cpp

```diff
@@ -709,8 +709,11 @@ ConvSolution ConvBinWinoRxS<Winodata, Winofilter>::GetSolution(
     kernel.l_wk.push_back(1);
     kernel.l_wk.push_back(1);
 
+    const auto force_cache_bypass = StartsWith(name, "gfx940") || StartsWith(name, "gfx941");
```

Comment:
[Recommendation] We do not need StartsWith when comparing for equality.
```suggestion
    const auto force_cache_bypass = (name == "gfx940") || (name == "gfx941");
```
[Q] So we expect both 940 and 941 require cache bypass?